language/evm/examples/sources/Table.move

```diff
@@ -0,0 +1,35 @@
+/// This module defines Table for EVM.
+module Evm::Table {
+    native struct Table<K, V> has store, copy, drop;
+
```

Comment:
I guess a table should not have copy and perhaps also not drop, because it is a potentially very large data structure.

Reply:
Done. Good point!